Sources/Classes/RSEventRepository.m

```diff
@@ -595,5 +606,20 @@ - (void) __prepareScreenRecorder {
 #endif
 }
 
+- (void) registerBackGroundTask {
+    [RSLogger logDebug:@"EventRepository: registerBackGroundTask: Registering for Background Mode"];
```

Comment:
This log will go to line no 614

Reply:
Oh, Thank you for pointing this out. 
Actually, this function should register a background task every time it is being called, but before doing that it should check if the existing task is still valid and if so invalidate it. So we should pull that logic out of the else block and this log shall be printing always. 
Made the changes and pushed them accordingly.